CHANGELOG.md

```diff
@@ -2,6 +2,14 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/) and [Keep a changelog](https://github.com/olivierlacan/keep-a-changelog).
 
+## [0.4.0] - 2018-12-328
+## Changed
+- [#44](https://github.com/pryorda/vmware_exporter/pull/44) Embrace more twisted machinery + allow concurrent scrapes
+## Fixed
+- [#46](https://github.com/pryorda/vmware_exporter/issues/46) Failure instance: Traceback: <class 'KeyError'>
```

Comment:
Sorry, maybe I missed something, but I'm how was this fixed?

Reply:
Discussing over in https://gitter.im/vmware_exporter/community. But line 375 where it checks power state should address this.